cms/templatetags/cms_tags.py

```diff
@@ -1058,7 +1061,7 @@ def render_tag(self, context, **kwargs):
         context.push()
         template = self.get_template(context, **kwargs)
         data = self.get_context(context, **kwargs)
-        data['content'] = mark_safe(kwargs['nodelist'].render(data))
```

Comment:
CMSEditableObjectAdd will only add the cms frontend editor markup, it does not render any user content thus we can just skip it